src/Windows/Avalonia.Win32/Win32Platform.cs

```diff
@@ -115,7 +116,8 @@ public static void Initialize(Win32PlatformOptions options)
                 .Bind<PlatformHotkeyConfiguration>().ToSingleton<PlatformHotkeyConfiguration>()
                 .Bind<IPlatformIconLoader>().ToConstant(s_instance)
                 .Bind<NonPumpingLockHelper.IHelperImpl>().ToConstant(new NonPumpingSyncContext.HelperImpl())
-                .Bind<IMountedVolumeInfoProvider>().ToConstant(new WindowsMountedVolumeInfoProvider());
+                .Bind<IMountedVolumeInfoProvider>().ToConstant(new WindowsMountedVolumeInfoProvider())
+                .Bind<IPlatformAccentColorProvider>().ToConstant(new AccentColorProvider() { UseSystemAccentColor = options.UseSystemAccentColor });
```

Comment:
We still need to check Windows OS version, so if user will set UseSystemAccentColors to true, his app won't crash on Win7. Our other platform specific options works in similar way.